### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by jarlehansen**

### CODE REVIEW

**Comment by jarlehansen** [comment]
@fsjovatsen @asgeirn can this be merged?

File: src/main/java/no/fint/testrunner/controller/AuthInitController.java [hunk 1]
```diff
@@ -64,10 +63,9 @@ public ResponseEntity clientNotFound(Exception e) {
         return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("Sorry, men vi finner ikke klienten du valgte!");
     }
 
-    @GetMapping("/clear/{orgName}")
+    @GetMapping("/clear/{orgName}") // TODO should be a POST request
```
Comment (asgeirn) [inline src/main/java/no/fint/testrunner/controller/AuthInitController.java:66]: Or perhaps a `DELETE` ?

**Review by asgeirn — APPROVED** [review]
Looks good!

**Comment by asgeirn** [comment]
@jarlehansen Ok to merge.

**Pull request merged**